test/app-luatest/console_debugger_session_test.lua

```diff
@@ -213,47 +313,65 @@ local function run_debug_session(cmdline, sequence, header)
             end
             repeat
                 result = trim(unescape(fh:read({ timeout = 0.5 })))
+                debuglog('stdout output:', result)
             until result ~= '' or result ~= clean_cmd
             if expected ~= '' then
                 t.assert_str_contains(result, expected, false)
             end
         end
         fh:close()
     end
+    debuglog('Delete temporary file ', tmpfile)
+    os.remove(tmpfile)
 end
 
 -- `tarantool -d debug-target.lua`
 g.test_debugger = function(cg)
-    local debug_target_script = path_to_script .. 'debug-target.lua'
-    local cmd = { TARANTOOL_PATH, '-d', debug_target_script }
     local scenario_name = cg.params.sequence
     t.assert(scenario_name)
-    t.assert(sequences[scenario_name])
-    run_debug_session(cmd, sequences[scenario_name], dbg_header)
+    local sequence = sequences[scenario_name]
+    t.assert(sequence)
+    local tmpfile = table.remove(sequence, 1)
+    t.assert(type(tmpfile) == 'string')
+
+    local cmd = { TARANTOOL_PATH, '-d', tmpfile }
+    run_debug_session(cmd, sequences[scenario_name], tmpfile, dbg_header)
 end
 
 local g_self = t.group('self-debug')
 
 local shorter_sequence = {
+        dbg_script [[
+    1| local dbg = require 'luadebug'
+    2| dbg()
+    3| local date = require 'datetime'
+    4|
+    5| local T = date.new{hour = 3, tzoffset = '+0300'}
+    6| print(T)
+    7|
+    8| os.exit(0)
```

Comment:
previous debug scripts doesn't contain whitespace between command and `|` and here there is a whitespace.
I suggest removing here or add to other scripts for consistency.

Reply:
Good point, and I've made them all look consistent via adding space before and after all separating bars.

Reply:
Like: 
```diff
--- a/test/app-luatest/console_debugger_session_test.lua
+++ b/test/app-luatest/console_debugger_session_test.lua
@@ -132,18 +132,18 @@ local sequences = {
     -- full, complex sequence
     full = {
         dbg_script [[
-     1|local date = require 'datetime'
-     2|
-     3|local T = date.new{hour = 3, tzoffset = '+0300'}
-     4| print(T)
-     5|
-     6|local fmt = '%Y-%m-%dT%H%M%z'
-     7|local S  = T:format(fmt)
-     8|print(S)
-     9|local T1 = date.parse(S, {format = fmt})
-    10|print(T1)
-    11|
-    12|os.exit(0)
+     1 | local date = require 'datetime'
+     2 |
+     3 | local T = date.new{hour = 3, tzoffset = '+0300'}
+     4 |  print(T)
+     5 |
+     6 | local fmt = '%Y-%m-%dT%H%M%z'
+     7 | local S  = T:format(fmt)
+     8 | print(S)
+     9 | local T1 = date.parse(S, {format = fmt})
+    10 | print(T1)
+    11 |
+    12 | os.exit(0)
 ]],
         { ['\t'] = '' }, -- \t is a special value for start
         { ['n'] = dbg_prompt },
@@ -179,18 +179,18 @@ local sequences = {
     -- partial sequence with successful breakpoints added
     bp_set_ok = {
         dbg_script [[
-     1|local date = require 'datetime'
-     2|
-     3|local T = date.new{hour = 3, tzoffset = '+0300'}
-     4|print(T)
-     5|
-     6|local fmt = '%Y-%m-%dT%H%M%z'
-     7|local S  = T:format(fmt)
-     8|print(S)
-     9|local T1 = date.parse(S, {format = fmt})
-    10|print(T1)
-    11|
-    12|os.exit(0)
+     1 | local date = require 'datetime'
+     2 |
+     3 | local T = date.new{hour = 3, tzoffset = '+0300'}
+     4 | print(T)
+     5 |
+     6 | local fmt = '%Y-%m-%dT%H%M%z'
+     7 | local S  = T:format(fmt)
+     8 | print(S)
+     9 | local T1 = date.parse(S, {format = fmt})
+    10 | print(T1)
+    11 |
+    12 | os.exit(0)
 ]],
         { ['\t'] = '' }, -- \t is a special value for start
         { ['b +9'] = dbg_prompt },
@@ -204,10 +204,10 @@ local sequences = {
     -- partial sequence with failed breakpoint addsitions
     bp_set_fail = {
         dbg_script [[
-    1|local date = require 'datetime'
-    2|local T = date.new{hour = 3, tzoffset = '+0300'}
-    3|print(T)
-    4|os.exit(0)
+    1 | local date = require 'datetime'
+    2 | local T = date.new{hour = 3, tzoffset = '+0300'}
+    3 | print(T)
+    4 | os.exit(0)
 ]],
         { ['\t'] = '' }, -- \t is a special value for start
         { ['b'] = 'expects argument, but none received' },
@@ -224,18 +224,18 @@ local sequences = {
     -- partial sequence with breakpoints additions and removals
     bp_delete_ok = {
         dbg_script [[
-     1|local date = require 'datetime'
-     2|
-     3|local T = date.new{hour = 3, tzoffset = '+0300'}
-     4|print(T)
-     5|
-     6|local fmt = '%Y-%m-%dT%H%M%z'
-     7|local S  = T:format(fmt)
-     8|print(S)
-     9|local T1 = date.parse(S, {format = fmt})
-    10|print(T1)
-    11|
-    12|os.exit(0)
+     1 | local date = require 'datetime'
+     2 |
+     3 | local T = date.new{hour = 3, tzoffset = '+0300'}
+     4 | print(T)
+     5 |
+     6 | local fmt = '%Y-%m-%dT%H%M%z'
+     7 | local S  = T:format(fmt)
+     8 | print(S)
+     9 | local T1 = date.parse(S, {format = fmt})
+    10 | print(T1)
+    11 |
+    12 | os.exit(0)
 ]],
         { ['\t'] = '' }, -- \t is a special value for start
         { ['b +7'] = '{file}:7' },
@@ -253,10 +253,10 @@ local sequences = {
     -- partial sequence with failed breakpoints removals
     bp_delete_fail = {
         dbg_script [[
-    1|local date = require 'datetime'
-    2|local T = date.new{hour = 3, tzoffset = '+0300'}
-    3|print(T)
-    4|os.exit(0)
+    1 | local date = require 'datetime'
+    2 | local T = date.new{hour = 3, tzoffset = '+0300'}
+    3 | print(T)
+    4 | os.exit(0)
 ]],
         { ['\t'] = '' }, -- \t is a special value for start
         { ['bd'] = 'expects argument, but none received' },
@@ -342,14 +342,14 @@ local g_self = t.group('self-debug')
 
 local shorter_sequence = {
         dbg_script [[
-    1| local dbg = require 'luadebug'
-    2| dbg()
-    3| local date = require 'datetime'
-    4|
-    5| local T = date.new{hour = 3, tzoffset = '+0300'}
-    6| print(T)
-    7|
-    8| os.exit(0)
+    1 | local dbg = require 'luadebug'
+    2 | dbg()
+    3 | local date = require 'datetime'
+    4 |
+    5 | local T = date.new{hour = 3, tzoffset = '+0300'}
+    6 | print(T)
+    7 |
+    8 | os.exit(0)
 ]],
     { ['\t'] = '' }, -- \t is a special value for start
     { ['n'] = dbg_prompt },
```